### DISCUSSION TITLE
DPA-1 OC20 Checkpoint

### DISCUSSION THREAD

**Discussion by owen-rett** [deepmodeling/deepmd-kit#2249] [Q&A] [answered]
Hello, was reading your DPA-1 paper [1], and wanted to try out the efficacy of model pre-training on my own dataset / problem. I was wondering if the checkpoints for the OC20 pre-trained model are available anywhere? If not, I can pre-train the model myself, but wanted to ask in case I can save a fair bit of GPU time. 

[1] Zhang, Duo, et al. "DPA-1: Pretraining of Attention-based Deep Potential Model for Molecular Simulation." arXiv preprint arXiv:2208.08236 (2022).

**Comment by iProzd** [comment, marked as answer]
Hi, we recently uploaded the trained model on [AIS-Square](https://www.aissquare.com/), you can find manual and directly download it [here](https://www.aissquare.com/models/detail?pageType=models&name=DPA_1_OC2M).

**Reply by owen-rett** [reply]
Thanks, I've been able to download it, and am beginning the fine-tuning process.
